documentation/Getting-started.md

```diff
@@ -299,6 +299,32 @@ module.exports = {
 }
 ```
 
+
+### Screen capture
+If screen capture is supported and appropriate hardware is attached, the video output of the DUT can be captured. For the testbot, this requires a compatible video capture device to be connected, that works with v4l2 and enumerates on the `/dev/video0 interface`.
+
+If that is the case, then capture can be started using the `Worker` class `capture()` method, for example:
+```js
+const Worker = this.require('common/worker');
+const worker = new Worker('DEVICE_TYPE_SLUG', this.getLogger())
+await worker.capture('start');
+```
+
+This will trigger video capture to start, and frames will be saved as `jpg` files to the shared volume at `/data/capture`. Capture will continue until stopped with:
```

Comment:
```suggestion
This will trigger video capture to start, and frames will be saved as `jpg` files in the `/data/capture` directory (which is a shared volume). Capture will continue until stopped with:
```